packages/altair-fastify-plugin/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import { getDistDirectory, renderAltair, RenderOptions } from 'altair-static';
+import fp from 'fastify-plugin';
+import fastifyStatic from 'fastify-static';
+
+import type { FastifyPluginCallback } from 'fastify';
+
+export interface AltairFastifyPluginOptions extends RenderOptions {
+  /**
+   * URL to set as the server endpoint.
+   *
+   * By default is `/graphql`
+   */
+  endpointURL?: string;
+  /**
+   * URL to be used as a base for relative URLs and hosting needed static files.
+   *
+   * By default is `/altair/`
+   */
+  baseUrl?: string;
+  /**
+   * Path in which Altair will be accesible.
+   *
+   * By default is `/altair`
+   */
+  path?: string;
+}
+
+const fastifyAltairPlugin: FastifyPluginCallback<AltairFastifyPluginOptions> = (
+  fastify,
+  { path, ...renderOptions } = {},
+  done
+) => {
+  if (renderOptions.baseURL == null) {
+    renderOptions.baseURL = '/altair/';
+  }
+  if (renderOptions.endpointURL == null) {
+    renderOptions.endpointURL = '/graphql';
+  }
+  if (path == null) {
```

Comment:
```suggestion
  if (path === null) {
```

Reply:
== null is the equivalent to identify `undefined` and `null`, this would case to effectively not be a default value

Reply:
A quick search about it https://www.tutorialrepublic.com/faq/how-to-determine-if-variable-is-undefined-or-null-in-javascript.php#:~:text=Answer%3A%20Use%20the%20equality%20operator%20(%20%3D%3D%20)&text=Whereas%2C%20the%20null%20is%20a,no%20yet%20assigned%20a%20value.

Reply:
https://www.ecma-international.org/ecma-262/5.1/#sec-11.9.3

Reply:
It's just another way of doing it and accepting null, and transforming it to the default, I can just change it to === undefined, but it doesn't matter in the end.

Reply:
== also identifies empty string and the number 0. It's too ambiguous and you should be able to achieve the same with strict equality. For identifying defaults, you can compare with `undefined`, not `null`.

Reply:
https://runkit.com/pabloszx/defaultequalnull

not true

Reply:
I guess I was wrong. That goes to show the ambiguity that comes from the abstract equality. It is better to be explicit in these kinds of cases to avoid issues later 😄

Reply:
@imolorhe no problem 👌